bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/handler/RadioThermostatHandler.java

```diff
@@ -235,22 +235,27 @@ private void scheduleClockSyncJob() {
      * Sync the thermostat's clock with the host system clock
      */
     private void syncThermostatClock() {
-        Calendar c = Calendar.getInstance();
+        boolean success = false;
+
+        // don't sync clock if override is on because it will reset temporary hold
+        if (rthermData.getThermostatData().getOverride() == null
```

Comment:
`rthermData.getThermostatData().getOverride()` could be stored in a local variable to avoid two calls when not null.

Reply:
fixed